1-bootstrap/README.md

```diff
@@ -22,6 +22,96 @@ Each pipeline has the following associated resources:
 
 ## Usage
 
+### Pre-requisites
+
+#### Secrets Project
+
+You will need a project with secret manager to store your git credentials, throughout the documentation this will be referenced as `$GIT_SECRET_PROJECT`.
```

Comment:
```suggestion
You will need a project with Secret Manager to store your Git credentials, throughout the documentation this will be referenced as `$GIT_SECRET_PROJECT`.
```